.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: node_js
+script: echo "It's alright!"
```

Comment:
I think you might have rebased your changes on another feature branch instead of develop for this to happen?